scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -282,22 +272,23 @@ public static BuildTargetResult CompileStage2(BuildTargetContext c)
                 return result;
             }
 
-            // Build projects that are packed in NuGet packages, but only on Windows
-            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (CurrentPlatform.IsWindows)
```

Comment:
yeah ended up putting this back the way it was, but check out TestTargets, am building the test packages to a list of frameworks